apps/rush-lib/src/cli/RushCommandLineParser.ts

```diff
@@ -246,6 +269,22 @@ export class RushCommandLineParser extends CommandLineParser {
     }
   }
 
+  private _validateCommandLineConfigCommand(command: CommandJson): void {
+    // There are some restrictions on the 'build' and 'rebuild' commands.
+    if (command.name !== 'build' && command.name !== 'rebuild') {
+      return;
+    }
+
+    if (command.commandKind === 'global') {
+      throw new Error(`${RushConstants.commandLineFilename} defines a command "${command.name}" using ` +
+        `the command kind "global". This command can only be designated as a command kind "bulk".`);
+    }
+    if (command.safeForSimultaneousRushProcesses) {
+      throw new Error(`${RushConstants.commandLineFilename} defines a command "${command.name}" using ` +
+        `"safeForSimultaneousRushProcesses=true". This configuration is not supported for "${command.name}".`);
```

Comment:
>is configuration is not su [](start = 52, length = 26)

why?

Reply:
My understanding is that `safeForSimultaneousRushProcesses` is intended for one Rush process to call a nested Rush process. Before this change, build and rebuild were hard-coded to set this value to false. This check enforces that assumption when a `command-line.json` config specifies build or rebuild tasks.